package.json

```diff
@@ -24,7 +24,7 @@
   "homepage": "https://github.com/Portchain/logacious#readme",
   "dependencies": {
     "ain2": "2.0.0",
-    "lodash": "^4.17.20",
```

Comment:
I think leaving the caret is fine. According to semver, lodash 4.17 establishes some API which should be preserved throughout the `^4.17` range. So we shouldn't really care if our app uses 4.17 or 4.18, as long as we have parity between QA and production (which is guaranteed by the app-level `package-lock.json`).

Of course specifying `"4.17"` is more "predictable", but if every package did this then we're back to the same situation as having `package.json` in libraries.